CI/doc/source/conf.py

```diff
@@ -0,0 +1,146 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import contextlib
+import os
+import shutil
+import sys
+from typing import List
+
+sys.path.insert(0, os.path.abspath("../.."))
+sys.setrecursionlimit(1500)
+
+# Move logos over to doc directory
+p = os.path.join("_static", "logos")
+if not os.path.exists(p):
+    os.mkdir("_static/logos")
+
+for filename in os.listdir(os.path.join("..", "..", "..", "logos")):

```

Comment:
This assumes logo files are in a logo directory at the root of the directory. This should be removed or the logo files put in that folder

Reply:
A folder called logos exists in the branch.